src/interpreter_funcs/voxelize.rs

```diff
@@ -158,13 +160,22 @@ impl Func for FuncVoxelize {
         log: &mut dyn FnMut(LogMessage),
     ) -> Result<Value, FuncError> {
         let mesh = args[0].unwrap_mesh();
-        let voxel_dimensions = args[1].unwrap_float3();
+        let voxel_dimensions = Vector3::from(args[1].unwrap_float3());
```

Comment:
The rest of the code is slightly nicer when the voxel dimensions are already stored in a Vector